projects/robots/robotis/darwin-op/protos/Darwin-op.proto

```diff
@@ -42,6 +43,19 @@ PROTO Darwin-op [
 {
   %{
     local has_jersey = fields.jersey.value
+    local enable_backlash = fields.enableBacklash.value
+    -- tune backlash on joints
+    local backlash_head = 0.04
```

Comment:
I believe we should put here the value taken from in the datasheet of the Darwin-OP servo. Is that correct?
Also, for convenience, we could have single backlash value, like:
```lua
local backlash = 0.04
local backlash_head = backlash
local backlash_neck = backlash
...
```
That would make it easier to adjust.